portfolio/src/main/java/com/google/sps/servlets/BusinessCardServlet.java

```diff
@@ -135,7 +126,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    * @throws IllegalArgumentException if the file was not an image
```

Comment:
Don't forget to update comments when changing the method signature.

Also, it would be good to mention that the blob is automatically deleted if an exception is thrown.